<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NBConverter</class>
 <widget class="QDialog" name="NBConverter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>209</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Number Base Converter</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icon.svg</normaloff>icon.svg</iconset>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QWidget" name="centralwidget" native="true">
     <widget class="QLineEdit" name="hexaLine">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>120</y>
        <width>221</width>
        <height>31</height>
       </rect>
      </property>
      <property name="mouseTracking">
       <bool>false</bool>
      </property>
      <property name="maxLength">
       <number>15</number>
      </property>
      <property name="readOnly">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="decLine">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>120</y>
        <width>221</width>
        <height>31</height>
       </rect>
      </property>
      <property name="maxLength">
       <number>16</number>
      </property>
      <property name="readOnly">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="octLine">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>40</y>
        <width>221</width>
        <height>31</height>
       </rect>
      </property>
      <property name="maxLength">
       <number>16</number>
      </property>
      <property name="readOnly">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="binLine">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>40</y>
        <width>221</width>
        <height>31</height>
       </rect>
      </property>
      <property name="inputMethodHints">
       <set>Qt::ImhPreferNumbers</set>
      </property>
      <property name="maxLength">
       <number>16</number>
      </property>
     </widget>
     <widget class="QPushButton" name="exitBut">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>80</y>
        <width>84</width>
        <height>34</height>
       </rect>
      </property>
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Exit</string>
      </property>
     </widget>
     <widget class="QLabel" name="binLabel">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>10</y>
        <width>221</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Binary</string>
      </property>
     </widget>
     <widget class="QLabel" name="decLabel">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>90</y>
        <width>221</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Decimal</string>
      </property>
     </widget>
     <widget class="QLabel" name="hexLabel">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>90</y>
        <width>221</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Hexadecimal</string>
      </property>
     </widget>
     <widget class="QLabel" name="octLabel">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>10</y>
        <width>221</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Octal</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
